bar-trender-front-end/src/views/Main.js

```diff
@@ -0,0 +1,163 @@
+// import React from "react";
+import React, { useEffect, useState } from 'react';
+
+// reactstrap components
+import {
+  Button,
+  Input,
+  InputGroupAddon,
+  InputGroupText,
+  InputGroup,
+  Container,
+  Row,
+  Col,
+} from "reactstrap";
+
+// core components
+import MainNavbar from "../components/Navbars/MainNavbar.js";
+import MainHeader from "../components/Headers/MainHeader.js";
+import DefaultFooter from "../components/Footers/DefaultFooter.js";
+import Carousel from "./index-sections/Carousel.js";
+import ModalSelectedElement from "../components/ModalSelectedElement.js";
+
+import List from "../components/List";
+import withListLoading from '../components/withListLoading';
+
+import image_left from '../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7122945)
```suggestion
import image_left from "../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png";
```